model-optimizer/mo/utils/schema.json

```diff
@@ -0,0 +1,128 @@
+{
+	"definitions": {},
+	"$schema": "http://json-schema.org/draft-07/schema#", 
+	"$id": "https://example.com/object1624983563.json", 
```

Comment:
it should be in a domain intel controls https://json-schema.org/understanding-json-schema/basics.html#declaring-a-unique-identifier e.g. docs.openvinotolkit.org

Also, you likely need a version - e.g. docs.openvinotolkit.org/schema/[latest | 2022.1]/model-optimizer-replacement-config.json

Reply:
id is optional attribute so we decided remove it.